setup/.env.example

```diff
@@ -1,31 +1,87 @@
-ADMIN_GROUP_NAME="JUS.AZ.S BAC TECH S.AZ.JUS" # Cloud admins: "JUS.AZ.S Tenant Owners / Propriétaires du locataire S.AZ.JUS"
-ACR_PUBLISHERS_GROUP_NAME="JUS.AZ.S BAC TECH S.AZ.JUS" # Should not be changed until continuous integration is implemented
-ENTRA_CLIENT_NAME="Otto (Sandbox)" # The name of the Entra app registration
-HOST_NAME_PREFIX=jus-otto-sandbox # The prefix used in the URL. (i.e. https://<HOST_NAME_PREFIX>.<LOCATION>.cloudapp.azure.com)
-
-# Tags
-APP_NAME=Otto # The name that will be used for cost tracking
-ENVIRONMENT=Sandbox # Cloud environment (e.g. Sandbox, Development, Production)
-LOCATION=canadacentral # Azure region (e.g. canadacentral, canadaeast)
-CLASSIFICATION=Unclassified # Data classification (e.g. Unclassified, Protected B)
-COST_CENTER="Business Analytics Center (CC 12031)" # Cost center for the application
-CRITICALITY=NonEssential # Criticality of the application (e.g. Essential, NonEssential)
-OWNER="Business Analytics Centre" # Owner of the application
-
-# Application
-DJANGO_ENV=DEV # Django environment (e.g. DEV, UAT, PROD)
-DJANGO_DEBUG=True # Django debug mode (e.g. True, False)
-
-# OpenAI Usage Capacity
+# Version number for this environment file
+ENV_VERSION=1.2
+
+
+### AZURE VARIABLES ###
+
+# Dictates the naming convention of resources (e.g. sandbox, dev, test, uat, prod)
+INTENDED_USE=sandbox
+
+# Cloud admins: "JUS.AZ.S Tenant Owners / Propriétaires du locataire S.AZ.JUS"
+ADMIN_GROUP_NAME="JUS.AZ.S BAC TECH S.AZ.JUS"
+
+# Should not be changed until continuous integration is implemented
+ACR_PUBLISHERS_GROUP_NAME="JUS.AZ.S BAC TECH S.AZ.JUS"
+
+# The name of the Entra app registration
+ENTRA_CLIENT_NAME="Otto (Sandbox)"
+
+
+### APPLICATION URL ###
+
+# Option 1: Custom Domain (Production)
+# Uncomment and set this for environments with a custom domain
+# SITE_URL=https://otto.justice.gc.ca
+
+# Option 2: Azure-provided Domain
+# Uncomment and set this for environments without a custom domain
+# Format: otto-<environment>-<uniqueidentifier>
+DNS_LABEL=otto-sandbox-abc123
+
+# Choose ONE option above and comment out the other
+
+
+### TAGS ###
+
+# The name that will be used for cost tracking
+APP_NAME=Otto 
+
+# Cloud environment (e.g. Sandbox, Development, Production)
+ENVIRONMENT=Sandbox
+
+# Azure region (e.g. canadacentral, canadaeast)
+LOCATION=canadacentral
+
+# Data classification (e.g. Unclassified, Protected B)
+CLASSIFICATION=Unclassified
+
+# Cost center for the application
+COST_CENTER="Business Analytics Center (CC 12031)"
+
+# Criticality of the application (e.g. Essential, NonEssential)
+CRITICALITY=NonEssential
+
+# Owner of the application
+OWNER="Business Analytics Centre"
+
+
+### APPLICATION VARIABLES ###
+
+# Django environment (e.g. DEV, UAT, PROD)
+DJANGO_ENV=DEV 
+
+# Django debug mode (e.g. True, False)
+DJANGO_DEBUG=True
+
+# Application admin (comma-separated list of UPNs)
+OTTO_ADMIN=jason.annable@justice.gc.ca,hamza.aitmohamed@justice.gc.ca,jules.kuehn@justice.gc.ca
```

Comment:
Can we leave off email addresses from the repo and provide an example of an email address?